Scripts/DCS-BIOS/lib/modules/aircraft_modules/Bf-109K-4.lua

```diff
@@ -0,0 +1,350 @@
+	local returnValue = (GetDevice(0):get_argument_value(30)) * 63 + 26
+	return returnValue
+end
+Bf_109K_4:defineIntegerFromGetter("PROPELLER_PITCH_DEG_VALUE", getPropPitch, 65000, "Gauge Values", "Propeller Pitch Degrees")
+
+local function getFuelLevel()
+	local returnValue = (GetDevice(0):get_argument_value(33)) * 400
+	return returnValue
+end
+Bf_109K_4:defineIntegerFromGetter("FUEL_LEVEL_VALUE", getFuelLevel, 65000, "Gauge Values", "Fuel Level Liters")
+
+local function getFuelPressure()
+	local returnValue = (GetDevice(0):get_argument_value(25)) * 20
+	return returnValue
+end
+Bf_109K_4:defineIntegerFromGetter("FUEL_PRESSURE_VALUE", getFuelPressure, 65000, "Gauge Values", "Fuel Pres kg/cm2 Read as X.X")
+
+local function getOilPressure()
+	local returnValue = (GetDevice(0):get_argument_value(26)) * 100
+	return returnValue
+end
+Bf_109K_4:defineIntegerFromGetter("OIL_PRESSURE_VALUE", getOilPressure, 65000, "Gauge Values", "Oil Pressure kg/cm2 Read as X.X or XX.X")
+
+local function getOilTemp()
+	local returnValue = (GetDevice(0):get_argument_value(28)) * 130
+	return returnValue
+end
+Bf_109K_4:defineIntegerFromGetter("OIL_TEMPERATURE_VALUE", getOilTemp, 65000, "Gauge Values", "Oil Temp °C")
+
+local function getCoolantTemp()
+	local returnValue = (GetDevice(0):get_argument_value(27)) * 130
+	return returnValue
+end
+Bf_109K_4:defineIntegerFromGetter("COOLANT_TEMPERATURE_VALUE", getCoolantTemp, 65000, "Gauge Values", "Coolant Temp °C")
+
+local function getOxygenPressure()
+	local returnValue = (GetDevice(0):get_argument_value(34)) * 250
+	return returnValue
+end
+Bf_109K_4:defineIntegerFromGetter("OXYGEN_PRESSURE_VALUE", getOxygenPressure, 65000, "Gauge Values", "Oxygen Pressure kg/cm2")
+
+local function getAirspeed()
+	local returnValue = (GetDevice(0):get_argument_value(2)) * 750
+	return returnValue
+end
+Bf_109K_4:defineIntegerFromGetter("AIRSPEED_VALUE", getAirspeed, 65000, "Gauge Values", "Airspeed km/h")
+
+local function getAltitude()
+	local returnValue = (GetDevice(0):get_argument_value(9)) * 9999
+	return returnValue
+end
+Bf_109K_4:defineIntegerFromGetter("ALTITUDE_VALUE", getAltitude, 65000, "Gauge Values", "Altitude Meters")
+
+local function getTachometer()
+	local returnValue = (GetDevice(0):get_argument_value(29)) * 3200 + 400
+	return returnValue
+end
+Bf_109K_4:defineIntegerFromGetter("TACHOMETER_VALUE", getTachometer, 65000, "Gauge Values", "Tachometer RPM")
+
+local function getBaro()
+	local returnValue = (GetDevice(0):get_argument_value(10)) * 120 + 920
+	return returnValue
+end
+Bf_109K_4:defineIntegerFromGetter("BAROMETRIC_PRESSURE_VALUE", getBaro, 65000, "Gauge Values", "Barometric Pressure mb")
+
+local function getHDG()
+	local returnValue = (GetDevice(0):get_argument_value(13)) * 360
+	return returnValue
+end
+Bf_109K_4:defineIntegerFromGetter("COMPASS_HEADING_VALUE", getHDG, 65000, "Gauge Values", "Compass Heading")
+
+--Externals
+Bf_109K_4:defineIntegerFromGetter("EXT_POSITION_LIGHT_LEFT", function()
```

Comment:
You can use Module:defineBitFromDrawArgument to clean up these external calls

Reply:
OK!